Go Security Issue

Run go install golang.org/x/vuln/cmd/govulncheck@latest
go: downloading golang.org/x/vuln v1.1.4
go: downloading golang.org/x/telemetry v0.0.0-20240522233618-39ace7a40ae7
go: downloading golang.org/x/sync v0.10.0
go: downloading golang.org/x/mod v0.22.0
go: downloading golang.org/x/tools v0.29.0
=== Symbol Results ===

Vulnerability #1: GO-2025-3849
    Incorrect results returned from Rows.Scan in database/sql
  More info: https://pkg.go.dev/vuln/GO-2025-3849
  Standard library
    Found in: database/sql@go1.24.5
    Fixed in: database/sql@go1.24.6
    Example traces found:
Error:       #1: internal/trino/client.go:172:22: trino.Client.ExecuteQuery calls sql.Rows.Scan

Your code is affected by 1 vulnerability from the Go standard library.
This scan found no other vulnerabilities in packages you import or modules you
require.
Use '-show verbose' for more details.

--- a/go.mod
+++ b/go.mod
@@ -1,6 +1,6 @@
 module github.com/tuannvm/mcp-trino
 
-go 1.24.2
+go 1.24.6
 
 require (
 	github.com/coreos/go-oidc/v3 v3.14.1

